app/src/main/java/com/example/kotlincalculator/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.kotlincalculator.mvp.model
+
+import com.example.kotlincalculator.utils.EMPTY_STRING
+
+class CalculatorModel (var firstOperand : String = EMPTY_STRING, var secondOperand : String = EMPTY_STRING, var operator : String = EMPTY_STRING, var result : String = EMPTY_STRING) {
```

Comment:
you will never use the constructor like this.. please put those variables inside the class

Reply:
done